runner/src/run.rs

```diff
@@ -86,10 +102,10 @@ fn launch_qemu(args: &RunArgs, miralis: PathBuf, firmware: PathBuf) -> ExitCode
         qemu_cmd.arg("-cpu").arg(cpu);
     }
     qemu_cmd
+        .arg("-m")
+        .arg("8G")
```

Comment:
We could extend the config to specify how much memory we want for QEMU. There is already a `[qemu]` table.